### PR TITLE
Order Item Meta Box Tweaks

### PR DESCRIPTION
**Pull request by mikejolley**
This PR restyles the meta box to make things a little clearer, and also adds some extra meta data for shipping methods if needed (#10574).

This is the meta box after styling:

2016-03-22 at 17 14

Rows can be clicked which reveal bulk actions:

https://cloudup.com/cSrhGp-GS7x

Rather than show striked out prices, actual discount amounts per line are shown:

2016-03-22 at 17 17

And refunds are shown in a similar way:

2016-03-22 at 17 18

@claudiosmweb @justinshreve @jameskoster

### CODE REVIEW

File: assets/js/admin/meta-boxes-order.js [hunk 1]
```diff
@@ -851,46 +857,49 @@ jQuery( function ( $ ) {
...
-				} else {
-					$( '#woocommerce-order-items' ).find( '.check-column input' ).removeAttr( 'checked' );
-				}
-			},
+		select_row: function() {
+			var $row = false;
+			if ( $( this ).is( 'tr' ) ) {
+				$row = $( this );
+			} else {
+				$row = $( this ).closest('tr');
```
Comment (justinshreve) [inline assets/js/admin/meta-boxes-order.js]: There are a few lines like this where there should be spacing between the parentheses and arguments.

File: includes/admin/meta-boxes/views/html-order-item.php [hunk 2]
```diff
@@ -7,48 +7,38 @@
  */
 
 if ( ! defined( 'ABSPATH' ) ) {
-	exit; // Exit if accessed directly
+	exit;
 }
+$product_link  = $_product ? admin_url( 'post.php?post                                                                                       =' . absint( $_product->id ) . '&action =edit' ) : '';
```
Comment (justinshreve) [inline includes/admin/meta-boxes/views/html-order-item.php]: Spacing seems off on these two lines (especially the huge gap in the admin url)

**Comment by justinshreve** [comment]
Code and result LGTM :+1:. Just noticed a few minor spacing things.

**Comment by claudiosmweb** [comment]
@thenbrent any chance of it break subscriptions? I'm asking because I know that you do some changes.

**Comment by thenbrent** [comment]
@claudiosmweb thanks the ping! I tested adding, editing and deleting line items on a subscription via the **Edit Subscription** screen and it all seems to be working well (no PHP or JS errors or notices). :+1:

**Pull request merged**